### PR TITLE
Make Repost a Modal

### PR DESCRIPTION
**Pull request by bradley-ray**
## Fixes

Issue Number:

## What is the current behavior?
Currently, links to create a repost takes a user to a seperate page.

## What is the new behavior?
Links to create a repost now open a modal. This made more sense considering most other actions are also done through modals. This behavior also exists in the LBRY app.

## PR Checklist

<details><summary>Toggle...</summary>

What kind of change does this PR introduce?

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

Please check all that apply to this PR using "x":

- [X] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [X] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

</details>

### CODE REVIEW

File: ui/component/fileActions/view.jsx [hunk 1]
```diff
@@ -120,15 +118,13 @@ export default function FileActions(props: Props) {
       <ClaimCollectionAddButton uri={uri} fileAction />
 
       {!hideRepost && !isMobile && !isLivestreamClaim && (
-        <Tooltip title={__('Repost')} arrow={false}>
+        <Tooltip title={__('Repost this claim')} arrow={false}>
```
Comment (infinite-persistence) [inline ui/component/fileActions/view.jsx:121]: In general, we are avoiding the use of "claim" in Odysee to simplify it from lbry-desktop.
I think the original should be ok?

File: ui/component/fileActions/view.jsx [hunk 2]
```diff
@@ -120,15 +118,13 @@ export default function FileActions(props: Props) {
...
           <Button
-            button="alt"
-            className="button--file-action"
+            button={'alt'}
+            className={'button--file-action'}
```
Comment (infinite-persistence) [inline ui/component/fileActions/view.jsx:124]: The original should be good -- use braces only when necessary.

**Review by infinite-persistence — CHANGES_REQUESTED** [review]

**Comment by tzarebczan** [comment]
Waiting on fixes from feedback in https://github.com/lbryio/lbry-desktop/pull/7491 also.

File: ui/component/channelAbout/view.jsx [hunk 3]
```diff
@@ -101,7 +101,7 @@ function ChannelAbout(props: Props) {
               amount={parseFloat(claim.amount) + parseFloat(claim.meta.support_amount)}
               precision={8}
             />{' '}
-            {SIMPLE_SITE && (
+            {!SIMPLE_SITE && (
```
[the remark this answers is not in the document]
Comment (bradley-ray) [reply ui/component/channelAbout/view.jsx:104]: I removed it to test something and forgot to add it back.

**Comment by infinite-persistence** [comment]
- [ ] The removal in the first comment has not been reverted.

File: ui/component/fileActions/view.jsx (hunk 2 above)
Comment (infinite-persistence) [reply ui/component/fileActions/view.jsx:124]: This one hasn't been reverted.

File: ui/modal/modalRepost/view.jsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
+// @flow
+import React from 'react';
+import { useHistory } from 'react-router';
+import Modal from 'modal/modal';
+import RepostCreate from 'component/repostCreate';
+import useThrottle from 'effects/use-throttle';
+
+type Props = {
+  uri: string,
+  name: string,
```
Comment (infinite-persistence) [inline ui/modal/modalRepost/view.jsx:10]: This should be `contentName`.  Lint is failing.

**Pull request merged**